apps/taskman/tests/test_integration.py

```diff
@@ -102,6 +102,24 @@ def test_task(self, monkeypatch, user_token, auth_client, test_api_uri):
         assert "assignee" in issue3["fields"]
         assert "customfield_12311140" in issue3["fields"]
 
+        response7 = auth_client.put(
+            f"{test_api_uri}/task/{issue['key']}/status",
+            data={"status": TaskStatus.CLOSED, "resolution": TaskResolution.DONE},
+            format="json",
+            **headers,
+        )
+        assert response7.status_code == 204
```

Comment:
I do not remember the exact details but I think that the Python bindings have some issues with an empty response... I wish @JakubFrejlach could explain it :smile:

Reply:
I have removed all 204 from taskman, returning the edited issue and changing for 200 status.